src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+.day {
+  position: relative;
+  width: $cell-size;
+  height: $cell-size;
+  background-color: #eee;
+  border: 1px solid #000;
+  transition: color 0.5s, transform 0.5s;
+
+  &:hover {
+    cursor: pointer;
```

Comment:
apply curson for normal state of element, not on hover

Reply:
Done